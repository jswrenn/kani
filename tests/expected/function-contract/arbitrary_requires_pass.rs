// Copyright Kani Contributors
// SPDX-License-Identifier: Apache-2.0 OR MIT

#[kani::requires(divisor != 0)]
fn div(dividend: u32, divisor: u32) -> u32 {
    dividend / divisor
}

#[kani::proof]
fn main() {
    div(kani::any(), kani::any());
}
