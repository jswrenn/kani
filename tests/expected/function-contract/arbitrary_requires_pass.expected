div.assertion \
	 - Status: SUCCESS \
	 - Description: "attempt to divide by zero" 

